packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -922,74 +950,81 @@ const ConstraintEditor = observer(
       editorStore.graphEditorMode.openElement(constraint._OWNER);
 
     return (
-      <div ref={ref} className="constraint-editor__container">
-        <PanelEntryDropZonePlaceholder
-          showPlaceholder={isBeingDragged}
-          className="constraint-editor__dnd__placeholder"
+      <PanelDnDEntry
+        ref={ref}
+        placeholder={
+          <div className="uml-element-editor__dnd__placeholder"></div>
```

Comment:
Let's do this next time


```suggestion
          <div className="uml-element-editor__dnd__placeholder"/>
```